entity-data-service-rx-client/src/main/java/org/hypertrace/entity/data/service/rxclient/EntityDataClient.java

```diff
@@ -23,13 +25,42 @@
    * @param entity
    * @return
    */
+  @Deprecated
   Single<Entity> getOrCreateEntity(Entity entity);
 
+  /**
+   * Performs a throttled update of the provided entity, starting after no longer than the provided
```

Comment:
I remember this one. Would an example be a good to further elaborate this?

Reply:
done